src/client/kotlin/tech/thatgravyboat/skyblockapi/api/profile/profile/ProfileAPI.kt

```diff
@@ -17,17 +19,30 @@ object ProfileAPI {
     // Profile: Watermelon ♲
     private val profileRegex = profileGroup.create(
         "name",
-        "Profile: (?<name>.+)"
+        "Profile: (?<name>.+)",
     )
 
+
     var profileName: String? = null
         private set
 
-    var profileType: ProfileType = ProfileType.UNKNOWN
-        private set
+    var profileType: ProfileType
+        private set(value) {
```

Comment:
remove the sets and just where profileType = is called just use the ProfileStorage.set and then make these vals